src/components/transaction-toast/components/transaction-toast-content/transaction-toast-content.tsx

```diff
@@ -0,0 +1,47 @@
+import { Component, EventEmitter, Prop, Event, h } from '@stencil/core';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import classNames from 'classnames';
+import { IToastDataState, ITransaction } from '../../transaction-toast.type';
+import { getIconHtmlFromIconLookup, getIconHtmlFromIconDefinition } from 'utils/getIconHtml';
+
+@Component({
+  tag: 'transaction-toast-content',
+  styleUrl: 'transaction-toast-content.css',
+  shadow: true,
+})
+export class TransactionToastContent {
+  @Prop() transactions: ITransaction[];
+  @Prop() toastDataState: IToastDataState;
+  @Prop() processedTransactionsStatus?: string | JSX.Element;
+  @Event() deleteToast: EventEmitter<void>;
+
+  render() {
+    const { icon, iconClassName, title, hasCloseButton } = this.toastDataState;
+
+    let iconHtml = null;
+    if (typeof icon === 'string') {
+      iconHtml = getIconHtmlFromIconLookup(icon);
+    }
+    if (icon instanceof HTMLElement) {
+      iconHtml = icon.outerHTML;
+    }
+    return (
+      <div class="content">
```

Comment:
checkout if there were any data testids in sdk-dapp

Reply:
done, testids added